oat-sa/tao-core
# Fix/nmc 61/large tree width

Related to : https://oat-sa.atlassian.net/browse/NMC-61
 
fix styles for item properties page with large trees
  
#### How to test
 
- Login as admin
- Go to Trees
- import large tree from files (added to Jira issue)
- Add a tree property to items class
- ensure tree widget fit to window width

#### Related PR's:
- https://github.com/oat-sa/tao-core-ui-fe/pull/194
